src/controller/python/README.md

```diff
@@ -359,239 +375,51 @@ example, `networkId=hex:0123456789abcdef` (for
-CHIP:CTL: Getting root certificate for the controller from the issuer
-CHIP:CTL: Generating credentials
-CHIP:CTL: Loaded credentials successfully
-CHIP:DL: Platform main loop started.
-Chip Device Controller Shell
+### `zcl ?`
 
-chip-device-ctrl >
-```
-
-The prompt `chip-device-ctrl >` indicates that the CHIP core library is loaded
-by Python, you can browse the symbols in the CHIP core library, setting
-breakpoints on functions and many other functions provided by GDB.
-
-You can use `Ctrl-C` to send SIGINT to the controller anytime you want so you
-can set breakpoints.
-
-> (`Ctrl-C` pressed here.)
+List available clusters.
 
 ```
-Thread 1 "python3" received signal SIGINT, Interrupt.
-0x0000fffff7db79ec in __GI___select (nfds=<optimized out>, readfds=0xffffffffe760, writefds=0x0, exceptfds=0x0, timeout=<optimized out>) at ../sysdeps/unix/sysv/linux/select.c:49
-49	../sysdeps/unix/sysv/linux/select.c: No such file or directory.
-(gdb)
+chip-device-ctrl > zcl ?
+dict_keys(['BarrierControl', 'Basic', 'ColorControl', 'DoorLock', 'Groups', 'IasZone', 'Identify', 'LevelControl', 'NetworkProvisioning', 'OnOff', 'Scenes', 'TemperatureMeasurement'])
 ```
 
-For example, you can break on `DeviceCommissioner::PairDevice` by using `break`
-command in GDB (`b` for short)
-
-```
-(gdb) b DeviceCommissioner::PairDevice
-Breakpoint 1 at 0xfffff5b0f6b4 (2 locations)
-(gdb)
-```
+### `zcl ? <Cluster>`
 
-Type `continue` (`c` for short) to continue the device controller, you may need
-another hit of `Enter` to see the prompt.
+List available commands in cluster. For example, for *LevlControl* cluster:
 
 ```
-(gdb) c
-Continuing.
-
-chip-device-ctrl >
+chip-device-ctrl > zcl ? LevelControl
+Move
+   moveMode: int, rate: int, optionMask: int, optionOverride: int
+MoveToLevel
+   level: int, transitionTime: int, optionMask: int, optionOverride: int
+MoveToLevelWithOnOff
+   level: int, transitionTime: int
+MoveWithOnOff
+   moveMode: int, rate: int
+Step
+   stepMode: int, stepSize: int, transitionTime: int, optionMask: int, optionOverride: int
+StepWithOnOff
+   stepMode: int, stepSize: int, transitionTime: int
+Stop
+   optionMask: int, optionOverride: int
+StopWithOnOff
+  <no arguments>
 ```
 
-Let do pairing over IP to see the effect of the breakpoint we just set.
+### `zclread <Cluster> <Attribute> <NodeId> <EndpointId> <GroupId> [arguments]`
 
-```
-chip-device-ctrl > connect -ip 192.168.50.5 20202021 1
-Device is assigned with nodeid = 1
+Read the value of ZCL attribute. For example:
 
-Thread 1 "python3" hit Breakpoint 1, 0x0000fffff5b0f6b4 in chip::Controller::DeviceCommissioner::PairDevice(unsigned long, chip::RendezvousParameters&)@plt ()
-   from /home/ubuntu/.local/lib/python3.9/site-packages/chip/_ChipDeviceCtrl.so
-(gdb)
 ```
-
-The `@plt` symbol means it is a symbol used by dynamic library loader, type `c`
-(for `continue`) and it will break on the real function.
-
+chip-device-ctrl > zclread Basic VendorName 1234 1 0
```

Comment:
Worth to mention that `zclread ? <Cluster>` syntax is also supported. E.g.:
```
chip-device-ctrl > zclread ? Basic
InteractionModelVersion
VendorName
VendorID
ProductName
ProductID
UserLabel
Location
HardwareVersion
HardwareVersionString
SoftwareVersion
SoftwareVersionString
ManufacturingDate
PartNumber
ProductURL
ProductLabel
SerialNumber
LocalConfigDisabled
ClusterRevision
```

Reply:
Thanks, it is mentioned in the commands list details below, but I will add it here as well.